Src/Orbiter/Vesselstatus.cpp

```diff
@@ -175,7 +175,7 @@ bool Vessel::ParseScenarioLine2 (char *line, void *status)
 			Base *base = ((Planet*)vs->rbody)->GetBase (trim_string(cbuf));
 			if (!base) {
 				char cerr[1024];
-				sprintf (cerr, "Scenario parse error for vessel %s: base '%s' not found on body '%s'.", name, trim_string(cbuf), ((Planet*)vs->rbody)->Name());
+				sprintf (cerr, "Scenario parse error for vessel %s: base '%s' not found on body '%s'.", name.c_str(), trim_string(cbuf), ((Planet*)vs->rbody)->Name());
```

Comment:
If you can wait for my PR to update c++ standard to c++20, then all non-path-related `sprintf`s can be replaced with [`std::format`](https://en.cppreference.com/w/cpp/utility/format).